src/test/java/ParserTest.java

```diff
@@ -0,0 +1,47 @@
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
```

Comment:
Maybe you could take a look at `ParserTest.java`? It has the same issue, and for the rest of the test classes too!